src/Hazelcast.Net.Tests/Configuration/ExceptionsTests.cs

```diff
@@ -33,12 +33,12 @@ public void ConfigurationExceptionConstructors()
             Assert.That(e.Message, Is.EqualTo("exception"));
             Assert.That(e.InnerException, Is.Not.Null);
             Assert.That(e.InnerException.Message, Is.EqualTo("bang"));
-
+#if !NET8_0_OR_GREATER
             e = e.SerializeAndDeSerialize();
```

Comment:
Is `ConfigurationException` Serializable? Is this a protocol defined exception?

Reply:
It's a Hazelcast exception. The `SerializeAndDeSerialize()` is .Net binary serializer. There is nothing to do with serialization actually. Our exception has `[Serializable]` attribute but I'm not sure about the purposes. If it is designed for internal purposes there is no usage. If it is there for .net diagnostic reporting, it is deprecated.